ambry-tools/src/main/java/com.github.ambry/tools/admin/Readme.md

```diff
@@ -39,39 +40,39 @@ This tool exposes some admin APIs to get more information for debugging purposes
 
 #### Dump Index
 ```java
```

Comment:
rather than using java as the language in the markup, could you use bash (as these are not really java code)